voice/bxml/verbs/record.md

```diff
@@ -0,0 +1,48 @@
+{% method %}
+## XML: `<Record>`
+The Record verb allows a segment of audio to be recorded during a call. At the end of the recording, a [Record Complete](../callbacks/recordComplete.md) event is generated.
+
+### Attributes
+
+| Attribute                    | Description                                                                                                                                                                                                            |
+|:-----------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| recordCompleteUrl            | (optional) URL to send the [Record Complete](../callbacks/recordComplete.md) event to and request new BXML.                                                                                                            |
```

Comment:
```suggestion
| recordCompleteUrl            | (optional) URL to send the [Record Complete](../callbacks/recordComplete.md) event to once it has ended. Accepts BXML.                                                                                                            |
```

Reply:
I'm just now realizing that we use `recordingAvailable` but also `recordComplete`. I'd like to change the latter to `recordingComplete` as well, but that's not exactly a small change.
I guess it makes sense to keep it as-is, since it's specifically for the `Record` verb and not other recordings (viz., those started by `StartRecording`)

Reply:
There was a discussion about it in the solution doc. In the end `recordComplete` was chosen.